### PR TITLE
[docs] add kube-rbac-proxy usage example

### PR DESCRIPTION
**Pull request by stratnevpy**
## Description
Add kube-rbac-proxy usage example

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Add `kube-rbac-proxy` usage example.
impact_level: low
```

### CODE REVIEW

File: modules/300-prometheus/docs/FAQ_RU.md [hunk 1]
```diff
@@ -129,6 +129,88 @@ spec:
 
 Для обеспечения безопасности настоятельно рекомендуем использовать **kube-rbac-proxy**.
 
+### Пример безопасного сбора метрик с приложения, расположенного вне кластера
+
+Требование: из кластера должен быть сетевой доступ до сервиса **kube-rbac-proxy**, запущенного на удалённом сервере, а с удалённого сервера должен быть доступен **kubernetes-api**.
+
+Предположим, что есть внешний сервер, который доступен через интернет и на котором работает **node-exporter**. По-умолчанию, он слушает порт 9100 и доступен на всех интерфейсах. Нам нужно сделать так, что бы доступ был контроллируемым. Начнём с подготовки некоторых объектов на стороне кластера.
```
Comment (z9r5) [inline modules/300-prometheus/docs/FAQ_RU.md:136]:
```suggestion
Предположим, что есть внешний сервер, который доступен через интернет и на котором работает **node-exporter**. По-умолчанию, он слушает порт 9100 и доступен на всех интерфейсах. Нам нужно сделать так, чтобы доступ был контролируемым. Начнём с подготовки некоторых объектов на стороне кластера.
```

**Review by z9r5 — DISMISSED** [review]

File: modules/300-prometheus/docs/FAQ.md [hunk 2]
```diff
@@ -132,6 +132,94 @@ spec:
...
+   - kind: ServiceAccount
+     name: prometheus-external-endpoint-server-01
+     namespace: d8-service-accounts
+   ```
+
+2. Generate `kubeconfig` for the created `ServiceAccount` ([example of kubeconfig generation for `ServiceAccount`](https://deckhouse.io/documentation/v1/modules/140-user-authz/usage.html#creating-a-serviceaccount-for-a-machine-and-granting-it-access)).
+
+3. Put the `kubeconfig` on the *remote server*. Later on you will need to specify the `kubeconfig` path to the `kube-rbac-proxy` (the example uses the path `${PWD}/.kube/config`).
+
+4. Configure `node-exporter` on the *remote server* to be accessible only on the local interface and listen to `127.0.0.1:9100`.
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:186]:
```suggestion
4. Configure `node-exporter` on the *remote server* to be accessible only on the local interface (i.e., listening on `127.0.0.1:9100`).
```

File: modules/300-prometheus/docs/FAQ.md [hunk 3]
```diff
@@ -132,6 +132,94 @@ spec:
...
+5. Run `kube-rbac-proxy` on the *remote server*:
+
+   ```shell
+   docker run --network host -d -v ${PWD}/.kube/config:/config quay.io/brancz/kube-rbac-proxy:v0.14.0 --secure-listen-address=0.0.0.0:8443 \
+     --upstream=http://127.0.0.1:9100 --kubeconfig=/config --logtostderr=true --v=10
+   ```
+
+6. Check, that port `8443` is accessible from the remote server's external address.
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:194]:
```suggestion
6. Check that port `8443` is accessible at the remote server's external address.
```

**Review by kirkonru — APPROVED** [review]

File: modules/300-prometheus/docs/FAQ.md [hunk 4]
```diff
@@ -132,6 +132,94 @@ spec:
...
+### An example of collecting metrics securely from an application outside a cluster
+
+Suppose there is a server exposed to the Internet on which the `node-exporter` is running. By default, the `node-exporter` listens on port `9100` and is available on all interfaces. One needs to ensure access control to the `node-exporter` so that metrics can be collected securely. Below is an example of how you can set this up.
+
+Requirements:
+- There must be network access from the cluster to the `kube-rbac-proxy` service running on the *remote server*.
+- The *remote server* must have access to the Kubernetes API server.
+
+Follow these steps:
+1. Create a new **ServiceAccount** with the following permissions:
```
Comment (z9r5) [inline modules/300-prometheus/docs/FAQ.md:144]:
```suggestion
1. Create a new `ServiceAccount` with the following permissions:
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4287](https://github.com/deckhouse/deckhouse/pull/4287) to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) successful!

**Pull request merged**